code/modules/hydroponics/grown/banana.dm

```diff
@@ -167,7 +167,12 @@
 	bite_consumption_mod = 4
 	var/is_ripening = FALSE
 
-/obj/item/food/grown/banana/bunch/Initialize(mapload, obj/item/seeds/new_seed)
+/obj/item/food/grown/banana/bunch/Initialize(
+		mapload,
+		/* starting_reagent_purity = */ null,
+		/* no_base_reagents = */ TRUE,
```

Comment:
```suggestion
		starting_reagent_purity = null,
		no_base_reagents = TRUE,
```
We use this pattern for when we are calling `new` on something because the named arguments do not pass properly from `atom/New` to `atom/Initialize`, but in this case this is actually the definition of an `Initialize` proc
Not only _can_ you name the arguments here, you actually _have_ to 😅